src/content/docs/apm/apm-ui-pages/monitoring/transactions-page-find-specific-performance-problems.mdx

```diff
@@ -48,9 +48,9 @@ The <DoNotTranslate>**Transactions**</DoNotTranslate> page also may include link
 
 ## Wall-clock time [#wall_clock_time]
 
-Wall-clock time is the amount of time recorded by the clock. New Relic uses wall-clock time for all the transactions and then sums that value across all of the transactions.
+Wall-clock time measures "real-time elapsed" during a specific transaction. For example, you're using a program to monitor transaction speeds. According to the program, the transaction took 30 seconds to complete, but one minute passed in the real world or on the wall clock due to discrepancies across the program. New Relic uses wall-clock time for all the transactions and then sums that value across all of the transactions.
 
-The host can execute requests in parallel, so you may see percentages over 100. For example, 100% would indicate that the execution time across all selected transactions is equal to the time expended when recording wall-clock time.
+The host can execute requests in parallel so that you may see percentages over 100. For example, 100% would indicate that the execution time across all selected transactions equals the time expended when recording wall-clock time.
```

Comment:
Not sure about the `,` > `that` change here; it seems to imply an intentionality. Perhaps instead `, which means` ? 

> The host can execute requests in parallel, which means you may see...

Reply:
Changed it back!